### PR TITLE
SBERDOMA-428 add missed status tranisition

### PR DESCRIPTION
**Pull request by Dimitreee**

### CODE REVIEW

**Review by pahaz — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]
Ok

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/organization/schema/Organization.js [hunk 1]
```diff
@@ -63,36 +63,30 @@ const Organization = new GQLListSchema('Organization', {
...
             schemaDoc: 'Graph of possible transitions for statuses. If there is no transition in this graph, ' +
                 'it is impossible to change status if the user in the role has the right to do so.',
-            type: Json,
-            defaultValue: DEFAULT_STATUS_TRANSITIONS,
-            hooks: {
-                validateInput: (args) => {
-                    return hasValidJsonStructure(args, true, 1, {})
-                },
+            type: Virtual,
+            graphQLReturnType: 'JSON',
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/Organization.js:67]: In a future, it's better to use more GQL friendly type %)

File: apps/condo/domains/organization/schema/Organization.js [hunk 2]
```diff
@@ -63,36 +63,30 @@ const Organization = new GQLListSchema('Organization', {
...
+            resolver: () => {
+                return DEFAULT_STATUS_TRANSITIONS
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/Organization.js:69]: Could you please make them dependent on the organization country?

File: apps/condo/domains/organization/schema/Organization.js [hunk 3]
```diff
@@ -63,36 +63,30 @@ const Organization = new GQLListSchema('Organization', {
...
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/Organization.js:83]: Could you please make them dependent on the organization country?

File: apps/condo/domains/organization/schema/Organization.test.js [hunk 4]
```diff
@@ -115,67 +115,67 @@ describe('Organization', () => {
...
+            //                 'type': 'mutation',
+            //                 'target': 'updateOrganization',
+            //                 'restrictedFields': [ 'statusTransitions' ],
+            //             },
+            //         })
+            //         expect(data).toEqual({ 'obj': null })
+            //     })
+            // })
+
+            // it('cannot update "defaultEmployeeRoleStatusTransitions"', async () => {
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/Organization.test.js:149]: it's better to use `it.skip`

**Review by pahaz — CHANGES_REQUESTED** [review]
You forgot about OrganizationEmployeeRole.statusTransitions

**Pull request merged**